Riot/Assets/fr.lproj/Vector.strings

```diff
@@ -1421,7 +1421,7 @@
 "side_menu_action_feedback" = "Remarques";
 "side_menu_action_help" = "Aide";
 "side_menu_action_settings" = "Paramètres";
-"side_menu_action_invite_friends" = "Inviter des amis";
+"side_menu_action_invite_friends" = "Inviter à rejoindre Tchap";
```

Comment:
I would prefer to not change this Vector string, and add a new one in Tchap.strings
This Vector string is related to another action hidden in Tchap, I don't want to depend on it

Reply:
Fixed in last commit